src/display.js

```diff
@@ -7,6 +7,7 @@ import REC2020 from "./spaces/rec2020.js";
 import P3 from "./spaces/p3.js";
 import Lab from "./spaces/lab.js";
 import sRGB from "./spaces/srgb.js";
+import Color from "./color.js";
```

Comment:
The functional interface should never reference the `Color` class.

Reply:
Good point. Should I change the signature of the function to only accept `PlainColorObject`, since that seems to be what the function is meant to handle?

Reply:
`serialize` calls `getColor` on the color so `ColorTypes` should be ok.

Reply:
Didn't realize that these lines also expect `PlainColorObject`:

<https://github.com/color-js/color.js/blob/c0a48bf283f7858cf589e81f5566a485299aa344/src/display.js#L49-L52>

So I just put the call at the beginning of the function instead.